sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -206,15 +206,20 @@ public virtual TokenCredential Credential
                 }
                 else
                 {
-                    _credential = new ClientSecretCredential(
-                        GetVariable("TENANT_ID"),
-                        GetVariable("CLIENT_ID"),
-                        GetVariable("CLIENT_SECRET"),
-                        new ClientSecretCredentialOptions()
-                        {
-                             AuthorityHost = new Uri(AuthorityHostUrl)
-                        }
-                    );
+                    var clientSecret = ClientSecret;
+                    if (string.IsNullOrWhiteSpace(clientSecret))
+                    {
+                        _credential = new DefaultAzureCredential();
```

Comment:
let's pass options to this with `ExcludeManagedIdentityCredential = true` That will eliminate any delays of trying to probe for the IMDS endpoint.